deploy/packages/serial-libraries.yaml

```diff
@@ -55,7 +55,7 @@ packages:
       - assimp ^boost@1.65.0
       - boost
       - brion
-      - deflect  ^boost@1.65.0
+      - deflect ~deflect-qt  ^boost@1.65.0
```

Comment:
LGTM. 

I just notice that the variant name should have been `qt` instead of `deflect-qt`.